spec/requests/api/user_sign_in_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::UsersController, type: :request do
+  let(:user) { create :user }
+  let(:parameter) {
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks.